<launch>
	<arg name="veh"/>
	<arg name="config" default="baseline"/>
	<arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman"/>
	<!-- <arg name="raw" default="true" doc="If set to true, also publish raw image using decoder."/>     -->
    <!--include file="$(find duckietown)/machines"/-->
	<group ns="$(arg veh)">
		<!--node machine="$(arg veh)" name="control_node" pkg="gd_motion" type="control_node_sequence.py" output="screen" clear_params="true" required="true">
		</node-->
		<node name="control_node" pkg="gd_motion" type="control_node_sequence.py" output="screen" clear_params="true" required="true">
		</node>
	</group>

	<!-- Publications -->


</launch>
